erc20-load-test-tool/script/.env

```diff
@@ -11,15 +11,15 @@ export WEB3J_ETHCLIENT_PORT=8545
 export WEB3J_ETHCLIENT_CORRELATE=true
 
 # WORKLOAD
-export WORKLOAD_TRANSACTIONS=1000
+export WORKLOAD_TRANSACTIONS=100
 export WORKLOAD_MODEL=CLOSED
 export WORKLOAD_LOAD_FACTOR=1
 
 # RECEIPT
-export WEB3J_RECEIPT_ATTEMPTS=0
+export WEB3J_RECEIPT_ATTEMPTS=1
```

Comment:
I see that lot of default values have changed in .env. Is it accidental or intentional?

Reply:
It is intentional, as discussed with @ramkri123